StoryboardType.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  StoryboardType.swift
+//  FuntastyKit
+//
+//  Created by Patrik Potoček on 20.12.16.
+//  Copyright © 2016 The Funtasty. All rights reserved.
```

Comment:
Sure, but this code is originally from Maarked where it is with your name and Funtasty copyright. Thus wrong *Created by*.